tester_app/client/app/bundles/HelloWorld/startup/helloWorldClientEntryComponent.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
This layer is not needed for the simple non-redux case.

We have:

1. Smart components (in containers)
2. Dumb components.

So let's get rid of this layer for non-redux.